nixos/modules/programs/steam.nix

```
@@ -4,52 +4,114 @@ with lib;
 
 let
   cfg = config.programs.steam;
+  gsCfg = config.programs.gamescope;
```

I think this should be a bit more descriptive since this derivation mostly concerns just steam itself
```suggestion
  gamescopeCfg = config.programs.gamescope;
```

---

done